README.md

```diff
@@ -1,12 +1,13 @@
 # Radicle
 
-A peer-to-peer stack for code collaboration
+A peer-to-peer stack for code collaboration.
 
-- peer-to-peer: The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up your own hosting, and no need to trust companies with your data.
-- Terminal-first: A Radicle project contains a git repository as well as the issues and proposals associated with that repo. All accessible from your terminal.
-- Programmable: Ever wanted to tweak your code collaboration service? With Radicle each unit of functionality –a machine– is its own litte P2P program, written in the Radicle language. You can change them to suit your needs, or create entirely new ones.
+- **peer-to-peer** : Radicle a decentralized, collaborative revision control system built on IPFS. Write open source software without giving your data to third parties or setting up a server.
```

Comment:
I’m not sure I’d call Radicle a revision control system. Radicle uses Git as its revision control system. Maybe “Radicle is a decentralized code collaboration tool...”

Reply:
I'll take what we land on for the homepage and update it here. But we agree with your comment.